src/pages/levelpack/Records.js

```diff
@@ -28,80 +33,180 @@ const Records = ({
   records,
   recordsLoading,
   showLegacyIcon,
+  showMoreStats,
+  levelStats,
 }) => {
   const [level, selectLevel] = useState(-1);
   const [longName, setLongName] = useState('');
   const [levelName, setLevelName] = useState('');
 
+  const { setShowMoreStats } = useStoreActions(state => state.LevelPack);
+
   if (recordsLoading) {
     return <Loading />;
   }
 
+  // eslint-disable-next-line no-unused-vars
+  const anyAreLegacy = hasSource(records);
```

Comment:
should probably delete or use it further down instead of reusing the function

Reply:
I think the reason I left this was because if i delete it then the function will be un-used but we might want to use the function in the future. The legacy icon now shows in the time column right beside the time so we don't need this parameter, which was previously used to have a variable number of columns in both the table header and table rows. Anyways i'll take a look but if I do it properly I might just end up deleting the function and if someone ever needs a similar function in the future they'll probably just re-write it because they won't know that it used to exist.

Reply:
removed var and the function it called